mxcube3/ui/components/Login.jsx

```diff
@@ -1,50 +1,103 @@
+'use strict';
 import ReactDOM from 'react-dom';
-import React from 'react';
+import React, { Component, PropTypes } from 'react'
+import { Redirect, Router, Route } from 'react-router';
 import classNames from 'classnames';
 import "bootstrap-webpack!bootstrap-webpack/bootstrap.config.js";
-import { Nav, Input, ButtonInput } from "react-bootstrap";
+import { Input, ButtonInput, Form, Well } from "react-bootstrap";
+import './Login.css';
+import Main_login from './Main_login'
+import SampleViewContainer from '../containers/SampleViewContainer'
```

Comment:
Why the Login component has to import SampleViewContainer ?
I think it is better if Login component is only for the login feature.
Moreover, this is a component not a container so I don't think it
should contain big parts of the application.